### PR TITLE
Milestone: Get started section

### PR DESCRIPTION
**Pull request by ForHemer**
This milestone add a **get started' section** to my portfolio. It contains:

- A contact form
- A list of social media to access.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -152,6 +152,33 @@ <h3>Skills</h3>
...
+      <section id="get-started">
+        <img class="get-started-img" src="images/image_geometry_1.svg" alt="image_geometry_3">
+        <div class="container">
+          <h2>Get started</h2>
+          <p class="get-started-content">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.
+        </p>
+        <form action="https://formspree.io/f/mlezegyq" method="POST">
+            <input type="text" id="name" name="user_name" maxlength="30" placeholder="Your name" required/>
+            <input type="email" id="email" name="user_email" placeholder="Your email" required/>
```
Comment (MrBrN197) [inline index.html:165]: - [x] You've implemented the features of the form well. Please ensure you are following the Figma design as closely as possible. If you notice the font you are using does not match the Figma
## Your Design Font
image

## Figma Design Font
image

The correct font to use is on the `input`s `Poppins`. you are currently using `Tahoma`.

Comment (MrBrN197) [inline index.html:165]: - [x] Please kindly try to match the Figma as closely as you can. notice that the "Start collaboration" button on the Figma design doesn't have a border. On your design you still have a border visible. please kindly remove it. 
## Figma Design
image

## Your Design 
image

**Review by MrBrN197 — APPROVED** [review]
## Hi, @ForHemer & @miftah1991  👋 

You've done a really nice job so far.
There are some issues that you still need to work on to go to the next project but you are almost there!
## To highlight 👏👏
- ✔Design matches well.
-  ✔ form Implemented correctly 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ForHemer) [reply index.html:165]: Hi @MrBrN197, I don't know what's happened with the button, but if you pay attention again to my CSS file you will see no border except when it 's pressed.

**Review by mateo951 — APPROVED** [review]
Hi @ForHemer and @miftah1991,

Great work making the changes requested by the previous reviewer  😎
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**